src/app/src/pages/eval/components/store.ts

```diff
@@ -285,12 +285,14 @@ export const useTableStore = create<TableState>()((set, get) => ({
             metric: computeAvailableMetrics(table),
             metadata: [],
             plugin: resultsFile.config?.redteam?.plugins?.map((plugin) => plugin.id) ?? [],
-            strategy: [
-              ...(resultsFile.config?.redteam?.strategies?.map((strategy) =>
-                typeof strategy === 'string' ? strategy : strategy.id,
-              ) ?? []),
-              'basic',
-            ],
+            strategy: Array.from(
```

Comment:
next time, consider making this a helper because it's duplicated 3 times in one file, strongly recommend preserving the order (or sorting)